code/game/machinery/autolathe/autolathe.dm

```diff
@@ -268,6 +268,10 @@
 		busy = TRUE
 		update_use_power(2)
 
+		for(var/mob/living/carbon/human/H in range (7, src))
+			if(H.is_hearing_sensitive())
```

Comment:
it might be worth making a global list of people listening intently rather than using a range loop, checking if they're hearing sensitive, then checking if they're listening intently. just check z level and distance on the human per loop